core-plugins/src/test/java/co/cask/hydrator/plugin/batch/sink/KVTableSinkTest.java

```diff
@@ -35,7 +39,8 @@ public void testTableSinkWithMissingKeyField() {
     KVTableSink.KVTableConfig kvTableConfig = new KVTableSink.KVTableConfig("purchases", "rowKey", "user");
     KVTableSink kvTableSink = new KVTableSink(kvTableConfig);
 
-    MockPipelineConfigurer mockPipelineConfigurer = new MockPipelineConfigurer(inputSchema);
+    MockPipelineConfigurer mockPipelineConfigurer = new MockPipelineConfigurer(inputSchema,
```

Comment:
There is a constructor that just takes the input schema right? If so, just use that one.